pkg/ansible/watches/watches.go

```diff
@@ -42,6 +47,9 @@ type Watch struct {
 	WatchDependentResources     bool                    `yaml:"watchDependentResources"`
 	WatchClusterScopedResources bool                    `yaml:"watchClusterScopedResources"`
 	Finalizer                   *Finalizer              `yaml:"finalizer"`
+
+	// Not configurable via watches.yaml
+	MaxWorkers       int `yaml:"maxWorkers"`
```

Comment:
I don't understand why watches has the `MaxWorkers`. Because looking through this PR, it seems it's a thing we keep passing around to the watches to force it to be on it when in reality it is some parameter that needs to get to the controller option.

Reply:
Either it's a command line argument applied to all watches, or it's an environment variable set to override a specific watch. Controller options are on a per watch basis. Is that not sufficient reason for the watch to hold that value?

Reply:
I'll let you decide if this is okay with you. I think having all of the logic in one place is best even though `maxWorkers` is weird.